### ISSUE TITLE
How to get user input in colang v2

### ISSUE THREAD

**Issue by nhantran0506** [NVIDIA/NeMo-Guardrails#716]
I wonder if there any ways so that I can get the user input in colang file?

**Comment by Pouyanpi** [comment]
Hi @nhantran0506, would you clarify what you intend to do?

Have you consulted the latest [documentation](https://docs.nvidia.com/nemo/guardrails/colang_2/overview.html)?

**Comment by drazvan** [comment]
@nhantran0506 : one way is to use the global variable `$last_user_message`. 

```colang
flow some name
  global $last_user_message

  # Do something with it
```

Note that there were some fixes related to this recently, so try on the `develop` branch. I'm not sure if it works in the latest published version (0.9.1.1 as of now). 

Alternatively, the flow can call an action, and if the action declares a parameter with the name `events`, it will get the complete events history and can extract the information from there.

**Comment by nhantran0506** [comment]
> [@nhantran0506](https://github.com/nhantran0506) : one way is to use the global variable `$last_user_message`.
> 
> ```
> flow some name
>   global $last_user_message
> 
>   # Do something with it
> ```
> 
> Note that there were some fixes related to this recently, so try on the `develop` branch. I'm not sure if it works in the latest published version (0.9.1.1 as of now).
> 
> Alternatively, the flow can call an action, and if the action declares a parameter with the name `events`, it will get the complete events history and can extract the information from there.

Thank @drazvan ,I will try on that now.
